packages/flutter/test/widget/bottom_sheet_test.dart

```diff
@@ -23,6 +23,7 @@ void main() {
     await tester.pump();
     expect(find.text('BottomSheet'), findsNothing);
 
+    // ignore: unawaited_futures
```

Comment:
rather than ignoring the future, we should capture it, and verify that it completes when appropriate